Lab4.ipynb

```diff
@@ -285,34 +285,41 @@
       "text/plain": [
-       "30.0"
+       "<Figure size 2000x1500 with 1 Axes>"
       ]
      },
-     "execution_count": 16,
      "metadata": {},
-     "output_type": "execute_result"
+     "output_type": "display_data"
     }
    ],
    "source": [
-    "np.max(counts)"
+    "fig, ax = plt.subplots(1, 1)\n",
+    "x = np.arange(poisson.ppf(0.01, mean),\n",
+    "              poisson.ppf(0.99, mean))\n",
+    "ax.plot(x, poisson.pmf(x, mean), 'bo', ms=8, label='poisson pmf')\n",
+    "ax.plot(x, poisson.pmf(x, mean), 'r-', ms=8, label='poisson pmf')\n",
+    "plt.yscale('log')\n",
+    "ax.vlines(x, 0, poisson.pmf(x, mean), colors='b', lw=5, alpha=0.5)\n",
+    "plt.show()"
```

Comment:
Could you also include the relationship that indicate how your Poisson distribution can be a model of the these data. For example, you can plot both the raw data and the Poisson distribution onto one single plot.